drivers/docker/driver_linux.go

```diff
@@ -1,9 +1,19 @@
 package docker
 
 import (
+	"strings"
+
 	"github.com/opencontainers/runc/libcontainer/cgroups"
 )
 
 func setCPUSetCgroup(path string, pid int) error {
-	return cgroups.WriteCgroupProc(path, pid)
+	// Sometimes the container exists before we can write the
+	// cgroup resulting in an error which can be ignored.
+	if err := cgroups.WriteCgroupProc(path, pid); err != nil {
```

Comment:
I don't recall the semantics of children processes and WriteCgroupProc.  Is there a risk that we may miss some children processes here?  Like if PID 1 is a entrypoint script and launches the main process without using `exec`, the main process may not be included in the cpusets?